src/NuGet.Core/NuGet.Shared/Extensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.Shared
+{
+    public static class Extensions
```

Comment:
internal

Maybe call this SharedExtensions or something that will not conflict anywhere else